code/script.js

```diff
@@ -1,9 +1,61 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
+const handleNameInput = (event) => {
```

Comment:
The parentheses around the argument are only needed when we have 0 or more than 1 argument (not when you have exactly 1 argument):
`const handleNameInput = event => {`